src/locale/README.md

```diff
@@ -52,6 +52,7 @@ Current supported languages:
 | Romanian                 | ro-RO        |
 | Spanish (Spain)          | es-ES        |
 | Turkish                  | tr-TR        |
+| Thai                     | th-TH        |
```

Comment:
`locale/README.zh-CN.md` should also be updated

Reply:
ok